azure/requests.yaml

```diff
@@ -3,6 +3,8 @@ releases:
   requests:
   - name: external-dns
     version: ">= 2.14.0"
+  - name: kube-state-metrics
```

Comment:
17.1.0 is already out, so by changing this block you are actually requiring a change in an existing release. 
That is why the CI is failing.

You need a new block for `> 17.1.0`